contiamo/data.py

```diff
@@ -1,13 +1,15 @@
 import datetime
 import logging
+import tempfile
+import warnings
+
 import numpy
 import pandas as pd
-import tempfile
 
 from contiamo.http_client import HTTPClient
 from contiamo.utils import contract_url_template_from_identifier, raise_response_error
 from contiamo.utils import get_file_extension
-from contiamo.errors import InvalidRequestError
+from contiamo.errors import ContiamoException, InvalidRequestError
```

Comment:
I know it's not part of this PR, but let's use this opportunity to alphabetize this import section, and we only need to import `from contiamo.utils` once.

Reply:
Good catch (the duplicate `utils` import).

Notice how I separated into standard imports, external imports, and internal ones already (with two out of three alphabetized!)

Reply:
yes, thank you!